docs/content/how-to/restartannotation.md

```diff
@@ -0,0 +1,18 @@
+# Restart Control Plane Components with Restart Annotation
```

Comment:
thanks! we also need to update docs/mkdocs.yml, see https://github.com/openshift/hypershift/pull/918/files

nit: Can we name this file restart-control-plane-components so it's consistent with what we do with the others how to sections?

Reply:
yea! Renamed and updated.